Kimjin/1주차 과제/picture.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>OST</title>
```

Comment:
ㅎㅎㅎㅎ 여기서 사용하는 h1 내용으로 바꿔야 할 것 같아요~
```suggestion
    <title>PICTURE</title>
```